### PR TITLE
changed format

### PR DESCRIPTION
**Pull request by kerct**

### CODE REVIEW

File: src/lib_ccx/asf_constants.h [hunk 1]
```diff
@@ -1,94 +1,119 @@
...
+  "\x9D\x8C\x17\x31\xE1\x03\x28\x45\xB5\x82\x3D\xF9\xDB\x22\xF5\x03"
+#define DVRMS_NTSC                                                             \
+  "\x80\xEA\x0A\x67\x82\x3A\xD0\x11\xB7\x9B\x00\xAA\x00\x37\x67\xA7"
+#define DVRMS_ATSC                                                             \
+  "\x89\x8A\x8B\xB8\x49\xB0\x80\x4C\xAD\xCF\x58\x98\x98\x5E\x22\xC1"
 // 10.13 - Undocumented DVR-MS properties
-#define DVRMS_PTS "\x2A\xC0\x3C\xFD\xDB\x06\xFA\x4C\x80\x1C\x72\x12\xD3\x87\x45\xE4"
+#define DVRMS_PTS                                                              \
+  "\x2A\xC0\x3C\xFD\xDB\x06\xFA\x4C\x80\x1C\x72\x12\xD3\x87\x45"               \
+  "\xE4" typedef struct
```
Comment (canihavesomecoffee) [inline src/lib_ccx/asf_constants.h]: Line is messed up

File: src/lib_ccx/ccx_common_common.h [hunk 2]
```diff
@@ -10,42 +10,44 @@
...
-#define EXIT_UNABLE_TO_DETERMINE_FILE_SIZE      6
-#define EXIT_MALFORMED_PARAMETER                7
-#define EXIT_READ_ERROR                         8
-#define EXIT_WITH_HELP                          9
-#define EXIT_NOT_CLASSIFIED                     300
-#define EXIT_ERROR_IN_CAPITALIZATION_FILE       501
-#define EXIT_BUFFER_FULL                        502
-#define EXIT_MISSING_ASF_HEADER                 1001
-#define EXIT_MISSING_RCWT_HEADER                1002
+#define EXIT_OK 0
```
Comment (canihavesomecoffee) [inline src/lib_ccx/ccx_common_common.h]: The old comments were actually more legible than the new ones... (imho)

File: src/lib_ccx/ccx_common_platform.h [hunk 3]
```diff
@@ -1,115 +1,115 @@
...
-			#define IN_MULTICAST(i)    IN_CLASSD(i)
-		#endif
-		#include <direct.h>
-		#define mkdir(path, mode) _mkdir(path)
-		#ifndef snprintf
-			// Added ifndef because VS2013 warns for macro redefinition.
-			#define snprintf(buf, len, fmt, ...) _snprintf(buf, len, fmt, __VA_ARGS__)
-		#endif
-		#define sleep(sec) Sleep((sec) * 1000)
+#ifdef _WIN32
```
Comment (canihavesomecoffee) [inline src/lib_ccx/ccx_common_platform.h]: This is less legible now...

**Comment by DanilaFe** [comment]
What is the general opinion on setting up something like a .clang-format file, and allowing a specialized tool to do the formatting? Seems like it would be the most consistent way to style code.

**Comment by canihavesomecoffee** [comment]
@DanilaFe That might be an option. Feel free to propose such a thing :)

Keep in mind it'll have to run (somehow) on windows too...

File: src/lib_ccx/ccx_common_platform.h (hunk 3 above)
Comment (kerct) [inline src/lib_ccx/ccx_common_platform.h]: which part?

**Pull request closed without merging**